build/Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM alpine:3.8
+#
+# Dockerfile for building local images.
+#
 
-RUN apk upgrade --update --no-cache
+# Build the manager binary
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 as builder
 
-USER nobody
+# Copy in the go src
+WORKDIR /go/src/github.com/kiegroup/kie-cloud-operator
+COPY . .
 
-ADD build/_output/bin/kie-cloud-operator /usr/local/bin/kie-cloud-operator
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o kie-cloud-operator github.com/kiegroup/kie-cloud-operator/cmd/manager
+
+# Copy the controller-manager into a thin image
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```

Comment:
this `FROM` should be from a minimal supported image? maybe rhel-atomic? worst case rhel7?

Reply:
You are already building on top of this image. see [here](https://github.com/openshift/release/blob/master/ci-operator/config/kiegroup/kie-cloud-operator/kiegroup-kie-cloud-operator-master.yaml#L10)

You don't need this.

Reply:
hmm, ok... is this a requirement @droslean ? that image is rather large... we'd like to keep our resulting deployable operator image under 75mb.

Reply:
I'll change it by fedora-minimal:30